### DISCUSSION TITLE
How to have actions upon hover?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#11004] [General - Components / Directives / etc] [answered]
I have a "delete" button that is expected to have three functions:

- action `delete` when clicking on it
- when hovering the button, having <something> popping up that has two possible actions:
  - a `delete after 2 days` button
  - an `erase` button

My first attempt was with QTooltip but it is really intended for tooltips and does not expand with the content (I tried to put a QCard inside).

I could also run a function at `@hover` and display a Qcard for instance, but I do not know how to place it near the button.

Is there a generic solution to such "more advanced tooltips"?

**Comment by hawkeye64** [comment, marked as answer]
@wsw70 Put QList inside of QMenu and call a ref on the QMenu to display/hide it
